pybamm/models/full_battery_models/base_battery_model.py

```diff
@@ -102,7 +102,7 @@ class BatteryModelOptions(pybamm.FuzzyDict):
                 reactions.
             * "open-circuit potential" : str
                 Sets the model for the open circuit potential. Can be "single"
-                (default), "current sigmoid", or "MSMR". If "MSMR" then the "particle"
+                (default), "current sigmoid", "DCHS", or "MSMR". If "MSMR" then the "particle"
```

Comment:
We should avoid acronyms that are not very common. Could we call this "Wysick" instead?

Reply:
I was just thinking along the lines of MSMR, and trying to find a name that captures the physics but not nessesarily the author. But I suppose I don't have any issues with calling it Wycisk instead.